src/test/scala/uk/gov/hmrc/api/specs/InboundPostAPIController.scala

```diff
@@ -58,7 +58,7 @@ class InboundPostAPIController extends BaseSpec, HttpClient:
         )
       )
       result.status shouldBe 400
-      val id             = (XML.loadString(body) \\ "IncludedBinaryObject").text.trim
+      val id             = (XML.loadString(body) \\ "ReceiveReferenceDataRequestResult").text.trim
```

Comment:
The API Platform team have confirmed that they will be replacing the tag content of the `ReceiveReferenceDataRequestResult`, not replacing the entire tag with another one, so we need to look for the UUID inside this element.